src/components/NewMeasure/CreateNewMeasureDialog.tsx

```diff
@@ -380,7 +391,18 @@ const CreateNewMeasureDialog = ({ open, onClose }) => {
             inputFormat="MM/dd/yyyy"
             value={formik.values.measurementPeriodEnd}
             onChange={(endDate) => {
-              formik.setFieldValue("measurementPeriodEnd", endDate);
+              const utcEndDate = new Date(
+                Date.UTC(
+                  endDate.getFullYear(),
+                  endDate.getMonth(),
+                  endDate.getDate(),
+                  0,
+                  0,
+                  0,
+                  0
```

Comment:
Based on the Slack discussion, shouldn't this be set to the end of the entered day?

Reply:
I forgot to save before committing...

Reply:
Oops, posted for review before I pushed